Patch Makefile so that nuttcp will build sanely on Debian systems.

Author: Chris Taylor <ctaylor@debian.org>
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,8 @@
-APP = nuttcp-6.1.2
-#EXTRAVERSION=-pre1
+DESTDIR=
+EXTRAVERSION=
+APP = nuttcp
 CC = gcc -Wall
-#OPT = -g -O0
-OPT = -O3
+OPT = -g -O2
 CFLAGS = $(OPT) $(NOIPV6)
 LIBS = 
 ifneq ($(NOIPV6),)
@@ -69,3 +69,10 @@ clean:
 tar:
 	(cd ..; $(TAR) cfj $(APP)$(EXTRAVERSION).tar.bz2 --exclude $(APP)/bin --exclude $(APP)/cygwin/*.bz2 --exclude $(APP)/rpm $(APP)$(EXTRAVERSION)$(APPEXT))
 	(cd ..; $(TAR) cfj $(APP)$(EXTRAVERSION).rpm.tar.bz2 --exclude $(APP)/bin --exclude $(APP)/cygwin  --exclude $(APP)/missing --exclude $(APP)/rpm $(APP)$(EXTRAVERSION)$(APPEXT))
+
+install:
+	mkdir -p ${DESTDIR}/usr/bin
+	install -m 0755 nuttcp ${DESTDIR}/usr/bin/nuttcp
+	mkdir -p ${DESTDIR}/usr/share/man/man8/
+	install -m 0644 nuttcp.8 ${DESTDIR}/usr/share/man/man8/
+
